csharp/ql/src/semmle/code/csharp/dataflow/DataFlow.qll

```diff
@@ -591,18 +581,47 @@ module DataFlow {
     abstract private class ReturnNode extends Node { }
```

Comment:
Could it be useful to have `getAReturnedExpr()` on this class?

Reply:
It is an internal class, so I think we should only add it if we need it later on.